app/src/main/java/io/intrepid/contest/screens/contestcreation/namecontest/NameContestPresenter.java

```diff
@@ -0,0 +1,28 @@
+package io.intrepid.contest.screens.contestcreation.namecontest;
+
+import android.support.annotation.NonNull;
+
+import io.intrepid.contest.base.BasePresenter;
+import io.intrepid.contest.base.PresenterConfiguration;
+import io.intrepid.contest.models.Contest;
+import io.intrepid.contest.models.ContestProvider;
+
+class NameContestPresenter extends BasePresenter<NameContestContract.View> implements NameContestContract.Presenter {
+
+    NameContestPresenter(@NonNull NameContestContract.View view,
+                         @NonNull PresenterConfiguration configuration) {
+        super(view, configuration);
+    }
+
+    @Override
+    public void onViewCreated() {
+        super.onViewCreated();
+    }
+
+    @Override
+    public void updateContestName(String contestName) {
```

Comment:
[Same] For me it makes more sense if presenter methods' names are more "passive" (e.g. "onContestNameUpdated"), so the view doesn't tell the presenter what to do.